changelog_unreleased/html/pr-8137.md

```diff
@@ -0,0 +1,24 @@
+#### Preserve normal space characters(like U+2005) ([#8137](https://github.com/prettier/prettier/pull/8137) by [@fisker](https://github.com/fisker))
+
+`U+2005` is not considered white-space in html, it should not be removed.
+
+```js
+// Prettier stable
+require("prettier")
+  .format("<i /> \u2005 <i />", { parser: "html" })
+  .slice(5, -6)
+  .split("")
+  .map((c) => c.charCodeAt(0).toString(16));
+
+// -> [ '20' ]
+// `U+2005` is removed
+
+// Prettier master
+require("prettier")
+  .format("<i /> \u2005 <i />", { parser: "html" })
+  .slice(5, -6)
+  .split("")
+  .map((c) => c.charCodeAt(0).toString(16));
+
+// -> [ '20', '2005', '20' ]
+```
```

Comment:
Is it okay to use js code show the difference?

Reply:
Totally okay.